azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_ops_iothub.py

```diff
@@ -80,19 +80,19 @@ class SendTelemetry(PipelineOperation):
 
     def __init__(self, message, callback=None):
         """
-        Initializer for SendTelemetry objects.
+        Initializer for SendTelemetryOperation objects.
 
         :param Message message: The message that we're sending to the service
         :param Function callback: The function that gets called when this operation is complete or has failed.
          The callback function must accept A PipelineOperation object which indicates the specific operation which
          has completed or failed.
         """
-        super(SendTelemetry, self).__init__(callback=callback)
+        super(SendTelemetryOperation, self).__init__(callback=callback)
         self.message = message
         self.needs_connection = True
 
 
-class SendOutputEvent(PipelineOperation):
+class SendOutputEventOperation(PipelineOperation):
```

Comment:
>SendOutputEventOperation [](start = 6, length = 24)

Probably shouldn't keep "Event" here. It's a Message.

What even is an Output Message vs. Telemetry? I think we should speak with Yi tomorrow.